tutorials/simulation/10_array_objs.py

```diff
@@ -224,39 +224,7 @@
 
 psd_ave = psd.mean(0)
 
-# map to `~mne.time_frequency.Spectrum` class and explore API
-
-
-def spectrum_from_array(
-    data: np.ndarray,  # spectral features
-    freqs: np.ndarray,  # frequencies
-    inst_info: mne.Info,  # the meta data of MNE instance
-) -> mne.time_frequency.Spectrum:  # Spectrum object
-    """Create MNE averaged power spectrum object from custom data"""
-    state = dict(
-        method="my_welch",
-        data=data,
-        sfreq=inst_info["sfreq"],
-        dims=("channel", "freq"),
-        freqs=freqs,
-        inst_type_str="Raw",
-        data_type="Averaged Power Spectrum",
-        info=inst_info,
-    )
-    defaults = dict(
-        method=None,
-        fmin=None,
-        fmax=None,
-        tmin=None,
-        tmax=None,
-        picks=None,
-        proj=None,
-        reject_by_annotation=None,
-        n_jobs=None,
-        verbose=None,
-    )
-    return mne.time_frequency.Spectrum(state, **defaults)
-
-
-spectrum = spectrum_from_array(data=psd_ave, freqs=freqs, inst_info=info)
+spectrum = mne.time_frequency.spectrum.SpectrumArray(
```

Comment:
Need to update the text in the preceding paragraph about how this is done / its experimental status. Also:

```suggestion
spectrum = mne.time_frequency.SpectrumArray(
```